ggml/src/ggml-cpu/CMakeLists.txt

```diff
@@ -211,8 +211,8 @@ function(ggml_add_cpu_backend_variant_impl tag_name)
                 list(APPEND ARCH_FLAGS /arch:AVX)
                 list(APPEND ARCH_DEFINITIONS GGML_AVX)
             else ()
-                list(APPEND ARCH_FLAGS /arch:SSE4.2)
```

Comment:
Did you test this?

Reply:
I did not test the windows part, sorry. I don’t use windows